File: management/src/main/java/io/hexaforce/management/controller/ExamVoucherPassRejectionController.java
package io.hexaforce.management.controller;

import java.util.Arrays;
import java.util.List;

import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.http.HttpStatus;
import org.springframework.http.ResponseEntity;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.ResponseBody;
import org.springframework.web.bind.annotation.RestController;

import io.hexaforce.dievas.commons.DievasServerMesseage;
import io.hexaforce.dievas.service.ExamVoucherPassRejectionService;
import lombok.extern.slf4j.Slf4j;

/**
 * ExamVoucherPassRejection router function.
 * 
 * @version 1.0.0.BUILD-SNAPSHOT
 * @author T.Tantaka 
 */

@Slf4j
@RestController
@RequestMapping("/api")
public class ExamVoucherPassRejectionController {
	
	@Autowired
	private ExamVoucherPassRejectionService examVoucherPassRejectionService;
	
	@ResponseBody
	@GetMapping(value = "/exam-voucher-pass-rejection")
	public ResponseEntity<List<DievasServerMesseage>> init() {
		log.info("Controller(java): init()");
		String x = examVoucherPassRejectionService.init();
		return new ResponseEntity<List<DievasServerMesseage>>(Arrays.asList(new DievasServerMesseage(x)),HttpStatus.OK);
	}
	
}

File: webapp/src/main/resources/static/controller/ExamVoucherPassRejectionController.js
'use strict';
/*******************************************************************************
 * ExamVoucherPassRejectionController
 ******************************************************************************/
app.controller('ExamVoucherPassRejectionController', ['ExamVoucherPassRejectionService', '$scope', function (ExamVoucherPassRejectionService, $scope) {
  
  $scope.init = function () {
    console.log('ExamVoucherPassRejectionController.js: init()');
    ExamVoucherPassRejectionService.init();
    $scope.serviceName = ExamVoucherPassRejectionService.getInitResult()[0].message;
  }
  
}]);